ts/utils/options.ts

```diff
@@ -14,3 +14,13 @@ export function areSetEqual<T>(a: Option<Set<T>>, b: Option<Set<T>>) {
   items.forEach(item => diff.delete(item));
   return diff.size === 0;
 }
+
+// Check if 2 option strings has the same value
+export function areStringsEqual(
```

Comment:
@valentinacala 
this function returns true if we pass these parameters:
`areStringsEqual(some("zz"),some("zzz"))`
We could use `===` instead of regex for whole string comparison.
Is it possibile to add a default parameter to the function `caseInsensitive=false`?
And at the end, could you write some tests?

Reply:
You're right.. updated! (https://github.com/teamdigitale/io-app/pull/1443/commits/c0b7d9629a3caeaf9e3faba5553cf649274ae96c)